docs/ignoring-files-folders-code.md

```diff
@@ -32,7 +32,32 @@ Without user customization, Semgrep refers to the following to define ignored fi
 In the absence of a user-generated `.semgrepignore`, Semgrep will refer to [its repository's default template](https://github.com/returntocorp/semgrep/blob/develop/semgrep/semgrep/templates/.semgrepignore):
 
 ```
```

Comment:
Unless the deploy process changed very recently, I think you can discard this change?

Reply:
Oh no. Yeah, this was an accident. Fixing.